src/main/java/net/findeasily/website/service/PropertyService.java

```diff
@@ -2,11 +2,17 @@
 
 import org.springframework.stereotype.Service;
 
+import net.findeasily.website.repository.UserRepository;
+import net.findeasily.website.repository.PropertyRepository;
 /**
  * @author Hao Zhou
  * @since 2018-08-08
  */
 @Service
 public class PropertyService {
-
+    private final PropertyRepository propertyRepository;
+    @Autowired
```

Comment:
You have to import
```
import org.springframework.beans.factory.annotation.Autowired;
```

BTW: I plan to have `property` merged into `listing` table now. 🤣 
So far, I already duplicated all `property` table fields into `listing` table. 
(it may be not a good idea, but it seems to make life easier)